hyperspy/drawing/image.py

```diff
@@ -306,6 +303,8 @@ def plot(self, data_function_kwargs={}, **kwargs):
         self.figure.canvas.draw()
 
     def _add_colorbar(self):
+        # Bug extend='min' or extend='both' and power law norm
```

Comment:
What's exactly the bug?

Reply:
It is difficult to reproduce in a minimal example, but it you use the `extend='both'` in the colorbar and changes gamma in the GUI, the ticks of the colorbar are going to be behave weirdly.